README.md

```diff
@@ -189,6 +189,76 @@ builder.AddSecretsManager(configurator: options =>
 
 ```
 
+## Configuration via appsettings.json
+Many options can be specified in a JSON file rather than hardcoded within the app. To do so, 
+the `ReadFromConfigSection` method should be called, like so:
```

Comment:
I would rather specify that many options can be specified using the Microsoft Configuration system without limiting to the JSON file. After all, the configuration values can be provided using other providers.